test/server/cards/01_SOR/upgrades/SmugglingCompartment.spec.ts

```diff
@@ -0,0 +1,51 @@
+describe('Smuggling Compartment', function() {
+    integration(function(contextRef) {
+        describe('Smuggling Compartment\'s on attack ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [{ card: 'atst', upgrades: ['smuggling-compartment'] }, { card: 'wampa', upgrades: ['smuggling-compartment'] }],
+                    },
+                    player2: {
+                        groundArena: ['snowspeeder']
+                    }
+                });
+            });
+
+            it('should ready a resource on attack', function () {
+                const { context } = contextRef;
+
+                context.player1.exhaustResources(2);
+
+                const exhaustedResourcesBeforeAbility = context.player1.countExhaustedResources();
```

Comment:
Don't really need this variable in this case since we know from the above that the number of exhausted resources is 2

Reply:
Good point!